examples/powercli/sdrs-recommendation-webexteams/handler/script.ps1

```diff
@@ -0,0 +1,37 @@
+# Process function Secrets passed in
+$VC_CONFIG_FILE = "/var/openfaas/secrets/vc-sdrs-config"
+$VC_CONFIG = (Get-Content -Raw -Path $VC_CONFIG_FILE | ConvertFrom-Json)
+if($env:function_debug -eq "true") {
+    Write-host "DEBUG: `"$VC_CONFIG`""
+}
+
+# Process payload sent from vCenter Server Event
+$json = $args | ConvertFrom-Json
+if($env:function_debug -eq "true") {
+    Write-Host "DEBUG: json=`"$($json | Format-List | Out-String)`""
+}
+
+$vcenter = ($json.source -replace "/sdk","")
+$datacenter = $json.data.datacenter.name
+$DatastoreCluster = $json.data.objectname
+
+$Body = "A new storage DRS recommendation has been generated `r`n"
+
+$Body = $Body + @"
```

Comment:
Is there more data in the incoming `com.vmware.vc.sdrs.StorageDrsNewRecommendationPendingEvent` that we could forward? Would it be possible to post an example of the input event and corresponding body received by Webex in the README?

Reply:
There's not a whole lot of extra data in the event that could be useful. All the relevant data is already included in the body being sent to Webex.

I've captured the output of an event and uploaded it to a [Gist](https://gist.github.com/mvandriessen/687b8071b995c27871799bef0e617480) so you can take a look. If you want, I'll still include it.

Reply:
I think it would still be useful to capture the output as part of the PR but agree there's not more useful detail